docs/tutorials/tutorial-sketches-theta.md

```diff
@@ -0,0 +1,327 @@
+
+Again, the set function is spliced in between the aggregator and the estimator.
+
+Likewise, use `THETA_SKETCH_UNION` to find the number of visitors that watched _any_ of the episodes:
+
+```sql
+SELECT THETA_SKETCH_ESTIMATE(
+         THETA_SKETCH_UNION(
+           DS_THETA(theta_uid) FILTER(WHERE "show" = 'Bridgerton' AND "episode" = 'S1E1'),
+           DS_THETA(theta_uid) FILTER(WHERE "show" = 'Bridgerton' AND "episode" = 'S1E2')
+         )
+       ) AS users
+FROM ts_tutorial
+```
+
+![Count distinct with Theta sketches, filters, and set operations](../assets/tutorial-theta-10.png)
+
+And finally, there is `THETA_SKETCH_NOT` which computes the set difference of two or more segments.
+The result describes how many visitors watched episode 1 of Bridgerton but not episode 2.
+
+
+```sql
+SELECT THETA_SKETCH_ESTIMATE(
+         THETA_SKETCH_NOT(
+           DS_THETA(theta_uid) FILTER(WHERE "show" = 'Bridgerton' AND "episode" = 'S1E1'),
+           DS_THETA(theta_uid) FILTER(WHERE "show" = 'Bridgerton' AND "episode" = 'S1E2')
+         )
+       ) AS users
+FROM ts_tutorial
+```
+
+![Count distinct with Theta sketches, filters, and set operations](../assets/tutorial-theta-11.png)
+
+## Conclusions
+
+- Counting distinct things for large data sets can be done with Theta sketches in Apache Druid.
+- This allows us to use rollup and discard the individual values, just retaining statistical approximations in the sketches.
+- With Theta sketch set operations, affinity analysis is easier, for example, to answer questions such as which segments correlate or overlap by how much.
+
+## Further reading
+
+See the following topics for more information:
+* [Theta sketch](../development/extensions-core/datasketches-theta.md) for reference on ingestion and native queries on Theta sketches in Druid.
+* [Theta sketch scalar functions](../querying/sql-scalar.md#theta-sketch-functions) and [Theta sketch aggregation functions](../querying/sql-aggregations.md#theta-sketch-functions) for Theta sketch functions in Druid SQL queries.
+* [Sketches for high cardinality columns](../ingestion/schema-design.md#sketches-for-high-cardinality-columns) for Druid schema design involving sketches.
+* [DataSketches extension](../development/extensions-core/datasketches-extension.md) for more information about the DataSketches extension in Druid as well as other available sketches.
+* The accuracy of queries using Theta sketches is governed by the size _k_ of the Theta sketch and by the operations you perform. See more details in the [Apache DataSketches documentation](https://datasketches.apache.org/docs/Theta/ThetaAccuracy.html).
+
+## Acknowledgments
+
+This tutorial is adapted from a blog post by Hellmar Becker. Visit the [original blog post](https://blog.hellmar-becker.de/2022/06/05/druid-data-cookbook-counting-unique-visitors-for-overlapping-segments/) on Hellmar Becker's blog.
```

Comment:
i've mixed opinions on this. On the one hand, it definitely seems a bit strange to me have this section in the docs that links out to another website, these docs are supposed to be the authority on Druid stuff if my mind. But on the other hand this is a pretty great content and it deserves credit.

Do we do stuff like this in other docs pages?

Do we do this anywhere else?

Reply:
We haven't done this on other docs pages. However, @hellmarbecker is a member of the Druid community and the originator of this content. We have some way to acknowledge that his authorship. Perhaps this would be cleaner:

This tutorial is adapted from a [blog post by Hellmar Becker](https://blog.hellmar-becker.de/2022/06/05/druid-data-cookbook-counting-unique-visitors-for-overlapping-segments/).

it also cuts down some of the repetition.

Reply:
I think the part that seemed strange is that it sort of seems vaguely like it is someone elses content, which isn't technically true if it is being (legally) contributed here.

Docs are contributed and licensed the same way as source files, and so this contribution is granting the ASF ownership of this content and licensing it to be distributed under the Apache license. The standard way we acknowledge external source contributions when necessary is the https://github.com/apache/druid/blob/master/LICENSE and https://github.com/apache/druid/blob/master/NOTICE files, though I'm unsure if it would be appropriate in this case, since the source packages are not the typical consumption method of the docs.

I just wanted to make sure that this is clear to @hellmarbecker to ensure that the license of this content is clean.

If all of this sounds ok, I do think it is ok to acknowledge and link to this external content since it was developed externally and later added instead of as an original Druid contribution (which we do not typically credit the author with, other than in the form of github commits and occasionally release announcements).

Reply:
Thanks, I am happy to grant permission for Apache to use the content!